src/ParameterTypes/passive_ren.jl

```diff
@@ -92,14 +99,29 @@ function direct_to_explicit(ps::PassiveRENParams{T}, return_h=false) where T
     # Currently converts to Hermitian to avoid numerical conditioning issues
```

Comment:
@MrstupidJ I've just noticed this comment here:
```julia
# Currently converts to Hermitian to avoid numerical conditioning issues
```
but we don't actually convert anything to a Hermitian matrix in the passive REN formulation. Is this just a rogue copy-paste job from the `GeneralRENParams` that can be delted?